.gitignore

```diff
@@ -0,0 +1 @@
+A test.md
```

Comment:
- [ ] Kindly Fix this so it says `test.md` not `A test.md` which ensures that if there's a file named "test.md", it would be ignored by git and not uploaded to the repository.
- [ ] Please also add `node_modules` to your `.gitignore` file.